charm/kafka/reactive/kafka.py

```diff
@@ -114,22 +97,20 @@ def waiting_for_certificates():
 )
 @when_not('kafka.started')
 def configure_kafka(zk):
-    if len(zk.zookeepers()) < 1:
-        hookenv.status_set('blocked', 'waiting for >= 1 zookeeper units')
-        return
-
     hookenv.status_set('maintenance', 'setting up kafka')
+
     log_dir = unitdata.kv().get('kafka.storage.log_dir')
     data_changed('kafka.storage.log_dir', log_dir)
-    kafka = Kafka()
+
     zks = zk.zookeepers()
-    kafka.configure_kafka(zks, log_dir=log_dir)
```

Comment:
I'm a little concerned that the charm logic is getting rewritten enough here that we might have broken something or lost functionality.

Reply:
@cmars It's the same log, I just change the function name to install just like zookeeper's